src/Components/Publishing/Byline/Share.tsx

```diff
@@ -120,6 +117,5 @@ const IconWrapper = styled.a`
   }
 `
 const ShareLabel = styled.span`
-  ${Fonts.unica("s16", "medium")}
-  margin: 10px 10px 10px 0px;
+  ${Fonts.unica("s16", "medium")} margin: 10px 10px 10px 0px;
 `
```

Comment:
This is maybe not fixable because of prettier, but kind of weird to have these on one line.

Reply:
It's fixable! Add a semicolon to the end of the interpolation: `${Fonts.unica("s16", "medium")};` I mentioned it in an earlier comment and said it's not necessary to address all of them in this PR.

Reply:
I'll fix this one though 😄 and any other ones I catch.